doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,207 @@ To send bare Lua code for execution, use ``Eval``:
+        -   No
+        -   Yes (structure and marshall functions must be predefined in Go code)
+
+    *   -   Access tuple fields by names
+        -   Only if marshalled to structure
+        -   No
+        -   Only if marshalled to structure
+
+    *   -   `SQL <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`_ support
+        -   No (`#62 <https://github.com/tarantool/go-tarantool/issues/62>`_), eval supported
+        -   No (`#18 <https://github.com/viciious/go-tarantool/issues/18>`_, closed), eval supported
+        -   No, eval supported
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`_
+        -   No (`#101 <https://github.com/tarantool/go-tarantool/issues/101>`_)
+        -   No
+        -   No
+
+    *   -   `Varbinary <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes (with in-built language tools)
+        -   Yes (with in-built language tools)
+        -   Questionable (decoding is weird with in-built language tools)
+
+    *   -   `UUID <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes
+        -   No
+        -   No
+
+    *   -   Decimal support
+        -   No (`#96 <https://github.com/tarantool/go-tarantool/issues/96>`_)
+        -   No
+        -   No
+
+    *   -   `EXT_ERROR <https://www.tarantool.io/ru/doc/latest/dev_guide/internals/msgpack_extensions/#the-error-type>`_
+            support
+        -   No
+        -   No
+        -   No
+
+    *   -   `[Soon] Datetime <https://github.com/tarantool/tarantool/discussions/6244>`_ support
+        -   No
+        -   No
+        -   No
+
+    *   -   `box.session.push() responses <https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_session/push/>`_
+        -   No (`#67 <https://github.com/tarantool/go-tarantool/issues/67>`_)
+        -   No (`#21 <https://github.com/viciious/go-tarantool/issues/21>`_)
+        -   Yes
+
+    *   -   `Session settings <https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/_session_settings/>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `[Soon] Graceful shutdown <https://github.com/tarantool/tarantool/issues/5924>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `IPROTO_ID (feature discovering) <https://github.com/tarantool/tarantool/issues/6253>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `tarantool/crud <https://github.com/tarantool/crud>`_ support
+        -   No
+        -   No
+        -   No
+
+    *   -   Support retrying
+        -   No
+        -   No
+        -   No
+
+    *   -   Implicit reconnecting strategy
+        -   Yes
+        -   Yes
+        -   Yes
+
+    *   -   Round robin failover
+        -   Yes
+        -   No
+        -   No
+
+    *   -   `[Soon] Master discovering <https://github.com/tarantool/tarantool/issues/6256>`__
+        -   No
+        -   No
+        -   No
```

Comment:
Added a link to the issue, but I don't get what "There is master discovering for all 1.10+" means. Do I need to change a link in header?

Reply:
Yeah, bad wording from my side. I meant that we plan to implement master discovery for 1.10+ in https://github.com/tarantool/go-tarantool/issues/113 and don't depend on https://github.com/tarantool/tarantool/issues/6256. However we likely will use built-in watchers as optimization (to avoid polling) for fresh tarantool versions.

Reply:
Since we don't assume 'master discovering using https://github.com/tarantool/tarantool/issues/6256', but rather 'master discovering' in the general sense, I would remove the link.

Reply:
Removed the link from "Master discovering"